Casks/a/ableset.rb

```diff
@@ -0,0 +1,33 @@
+cask "ableset" do
+  arch arm: "aarch64", intel: "x86_64"
+  folder = on_arch_conditional arm: "-arm64", intel: ""
```

Comment:
```suggestion
  arch arm: "-arm64"
```

We don’t need to specify the `intel:` parameter if we are setting it to an empty string.

Reply:
Changed in latest push